src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -20,7 +21,8 @@ class Decimal extends React.Component {
     this.state = {
       value,
       visibleValue: this.formatValue(value),
-      isControlled
+      isControlled,
+      isWiggling: false
```

Comment:
What about changing the name from `isWiggling` animation to `isAnimating`? It will be more flexible for the future if anyone will want to change the animation behavior